torchtune/modules/loss/cross_entropy_loss.py

```diff
@@ -143,3 +143,95 @@ def forward(
             return total_loss
         else:
             return total_loss / total_elements
+
+class LigerFusedCrossEntropyLoss(nn.Module, SFTLoss):
+    """Memory efficient Cross-entropy loss that uses fused CUDA kernels to compute the loss.
```

Comment:
I think this should mention that this is an approximation of CrossEntropyLoss and as such will have some numerical differences. Further down there should also be a "Note" that using this loss requires pip install triton, liger_kernel.